compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -234,7 +234,7 @@ genLoadLongConstant(TR::CodeGenerator * cg, TR::Node * node, int64_t value, TR::
    //disable LARL for AOT for easier relocation
    //cannot safely generate LARL for longs on 31-bit
    else if (cg->canUseRelativeLongInstructions(value) &&
-            comp->target().is64Bit() && !(comp->compileRelocatableCode()))
+            comp->target().is64Bit() && !(comp->compileRelocatableCode() || comp->isOutOfProcessCompilation()))
```

Comment:
The `comp->isOutOfProcessCompilation()` here will need to be replaced with something more appropriate.

Reply:
@dchopra001 you're right, that is the correct solution. I would override `canUseRelativeLongInstructions` in OpenJ9 and return `false` if we are a remote compile.

Reply:
It seems like this `isOutOfProcessCompilation()` now exists in OMR:
https://github.com/eclipse/omr/blob/ff6b4b2f91eab5054d0644d76a20f98ba168b4c6/compiler/compile/OMRCompilation.hpp#L1054-L1060

So we have the option of leaving this code as is and things will still work.  We can still override `canUseRelativeLongInstructions` if you prefer that approach. Let me know.

Reply:
This is not the only place that uses relative long instructions, i.e. there are other locations querying `canUseRelativeLongInstructions`, so my suggestion is to override this API in OpenJ9 and return false if we are a remote compile.

Reply:
Check removed here: aabd9acc9738af958cff3ae5ec9ea9e6206ddb8e.
Related OpenJ9 PR: https://github.com/eclipse/openj9/pull/10113